### ISSUE TITLE
variables5: shadowing solution should be more prominent

### ISSUE THREAD

**Issue by medicalwei** [rust-lang/rustlings#375]
For people coming from different programming languages that does not allow shadowing in the same scope, the natural solution for them is to change the assignments to the same type in order to compile.

**Comment by alexxroche** [comment]
I agree that shadowing isn't found in every language and no one could be expected to guess that shadowing is the solution. The rustlings Readme.md does link to The Rust Book which has in Chapter 3  [Shadowing](https://doc.rust-lang.org/stable/book/ch03-01-variables-and-mutability.html?highlight=shadow#shadowing)  (Shadowing is even introduced in Chapter 2.)

`rustlings hint variables5` also mentions Shadowing. Do you have any suggestions how to make the solution more prominent?

**Comment by YJDoc2** [comment]
Hello
What if instead of directly printing both number variables, we convert first to "three" (text, instead of digit) and call to_ascii_uppercase() or some &str specific function on it? It will force to keep type of first to &str. Similarly we can print (number + something) (for second case) so that second must be of numerical type.
Another way to force keeping types would be defining dummy functions above which takes specific types, and return strings, and print those being called on the variables, instead of directly printing, but it may be hard to understand as it is in variables exercise, which is supposed to be in the very start.

**Comment by alexxroche** [comment]
I think that a pull request would be helpful to demonstrate your idea and how it related to this ticket about exercise "_variables5: shadowing solution should be more prominent_". Thank you. (And if it isn't related it could still be beneficial.)

**Comment by YJDoc2** [comment]
Thanks :+1: 
I will implement the changes I was suggesting and make a PR according to CONTRIBUTING.md

**Comment by YJDoc2** [comment]
Hey, When running cargo test without any chages done to the repository gives this output :
```
running 4 tests
test exercise::test::test_finished_exercise ... ok
test exercise::test::test_pending_state ... ok
test exercise::test::test_clean ... ok
test exercise::test::test_exercise_with_output ... FAILED

failures:

---- exercise::test::test_exercise_with_output stdout ----
thread 'exercise::test::test_exercise_with_output' panicked at 'Failed to run 'run' command: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/exercise.rs:172:19
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    exercise::test::test_exercise_with_output

test result: FAILED. 3 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out

error: test failed, to rerun pass '--bin rustlings'
```

This is from exercise.rs, a file which I am not changing, but what is the reason for this?

**Comment by alexxroche** [comment]
That is a bug YJDoc2. I've opened a new ticket because this one is about variables5: shadowing.

**Comment by YJDoc2** [comment]
Hey 
For the commit message I'm adding `fix(variables5) : make shadowing more prominent`, should I also add `closes #375` ?
